hr_utilization_report/models/hr_department.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, api
+
+
+class HrDepartment(models.Model):
+    _inherit = 'hr.department'
+
+    @api.multi
+    def action_utilization_report_wizard(self):
```

Comment:
Isn't better to define this as an action window directly in XML for avoiding code?

Reply:
Why putting code directly in XML is better than calling a method from XML? At least latter one can be tested

Reply:
Any testing change can be done directly in UI. You can include in test both. Just `read` the action window instead of calling the method. Any way, this is not needed, as action window are already tested. The only possible missing thing is the parameter, which is correct if you put `active_ids`

Reply:
You win :) Updated the PR